scripts/scripts.js

```diff
@@ -455,12 +455,6 @@ async function loadLazy(doc) {
   autolinkModals(doc);
   const main = doc.querySelector('main');
   await loadBlocks(main);
-
```

Comment:
I don't understand how this change is related to the associated issue.

Reply:
Yes, you are right change is not related to the Issue. Without this change PR reports the poor LHS